### PR TITLE
Add orbit dump entries for systems

### PR DESCRIPTION
**Pull request by dbenage-cx**
Adds orbit entries for system debug dumps.

/cc @Vezzra

### CODE REVIEW

File: universe/Planet.cpp [hunk 1]
```diff
@@ -184,13 +184,15 @@ UniverseObjectType Planet::ObjectType() const
 { return OBJ_PLANET; }
 
 std::string Planet::Dump() const {
+    TemporaryPtr<System> cur_system = GetSystem(SystemID());
```
Comment (geoffthemedio) [inline universe/Planet.cpp]: If you have to get another object to get some info, it's probably not something that should be in a Dump function. The idea is to output this object's state, not any related information from other objects.

Comment (dbenage-cx) [reply universe/Planet.cpp]: Thank you, dropping planet dump and reworking system to report each orbit.

**Comment by dbenage-cx** [comment]
Testing with #869

**Comment by geoffthemedio** [comment]
looks ok... assuming you did some basic testing

**Comment by dbenage-cx** [comment]
@geoffthemedio sorry, i meant I was still testing.  Minor issue with the trailing `,`.
(Note: with current master, systems will report no orbits)

**Comment by geoffthemedio** [comment]
fix it and make another pull request then...

**Comment by Vezzra** [comment]
@adrianbroher: Not related to this PR per se, just taking it as an example: what category to use for this? It's not a refactoring or bug, and to label this as feature doesn't really fit IMO. It's more something like an extension/enhancement of an already existing thing, in this case the `Dump` function of the `System` class.

Personally I'd prefer to use "enhancement" for this kind of things. That ok with you, or would you prefer something else?

**Comment by Vezzra** [comment]
@dbenage-cx: Thanks! That should help with the debugging.

**Comment by adrianbroher** [comment]
@Vezzra we should keep that discussion in the corresponding thread in the forum.

http://www.freeorion.org/forum/viewtopic.php?f=4&t=10199

**Pull request merged**